deploy/install.sh

```diff
@@ -207,14 +257,24 @@ oc apply -f $KABANERO_SUBSCRIPTIONS_YAML --selector kabanero.io/install=13-subsc
 checksub openshift-pipelines openshift-operators
 checksub appsody-operator-certified openshift-operators
 
-# Install 14-subscription (che, kabanero)
+# Install 14-subscription (codeready-workspaces, kabanero)
+
+# Prior to installing codeready-workspaces, remove all eclipse-che instances and 
+# eclipse-che operator deployments in the kabanero namespace.
+removeResourceInstance checlusters.org.eclipse.che CheCluster kabanero
+unsubscribe eclipse-che kabanero
+
+# Codewind is required to run as privileged and as root because it builds container images
+oc adm policy add-scc-to-user anyuid system:serviceaccount:kabanero:che-workspace
+oc adm policy add-scc-to-user privileged system:serviceaccount:kabanero:che-workspace
```

Comment:
Is there something fundamentally different about using CRW instead of Che, with regards to these two `oc adm` commands?  Why didn't we have to do this when we were using the Che operator?

Reply:
Per the codewind documentation (https://www.eclipse.org/codewind/mdt-che-installinfo.html), setting the adm policies are/were required from a codewind point of view. It was just not done as part of the che installation, and left to the user to do that. Since this is explicitly required now, perhaps it should have been done for che.